samples/StaticHtmlComposites/package.json

```diff
@@ -4,9 +4,9 @@
   "description": "Basic bundle for communication react composites and usage on a static HTML page.",
   "private": true,
   "scripts": {
-    "build": "webpack",
+    "build": "webpack --config webpack.build.config.js",
```

Comment:
Instead of whole file for setting mode production, can just use `--mode=production` CLI arg instead here